server/chat-plugins/hangman.ts

```diff
@@ -37,7 +37,7 @@ try {
 	if (save) {
 		FS(HANGMAN_FILE).writeUpdate(() => JSON.stringify(hangmanData));
 	}
-} catch (e) {}
+} catch (e: any) {}
```

Comment:
Why does _this_ need a cast?

Reply:
The general response to this is that I did a mass find and replace. Might be nice to keep it as a general convention?

Reply:
I pushed a commit to remove the types from all patterns of `catch (e) {}`.